mobly/base_test.py

```diff
@@ -46,6 +46,30 @@ class Error(Exception):
     """Raised for exceptions that occured in BaseTestClass."""
 
 
+def _verify_controller_module(module):
+    """Verifies a module object follows the required interface for
+    controllers.
+
+    Args:
+        module: An object that is a controller module. This is usually
+            imported with import statements or loaded by importlib.
+
+    Raises:
+        ControllerError: if the module does not match the Mobly controller
+            interface, or one of the required members is null.
+    """
+    required_attributes = ('create', 'destroy', 'MOBLY_CONTROLLER_CONFIG_NAME')
+    for attr in required_attributes:
+        if not hasattr(module, attr):
+            raise signals.ControllerError(
+                'Module %s missing required controller module attribute'
+                ' %s.' % (module.__name__, attr))
+        if not getattr(module, attr):
```

Comment:
Should we check if `callable(getattr(module, 'create'))`, `callable(getattr(module, 'destroy'))`, and `isinstance(getattr(module, 'MOBLY_CONTROLLER_CONFIG_NAME'), str)`?

Reply:
A bit of an overkill?
If it's not callable, it'll fail anyway. The python "if it quacks like a duck" principle :P